build.sh

```diff
@@ -514,4 +514,28 @@ EOF
   # Copy dynamic loadable modules
   module_dir=$(printf '%s\n' $TARGET/lib/vips-modules-* | sort -n | tail -1)
   [ -d "$module_dir" ] && cp $module_dir/* $SOURCE_DIR/lib/ || true
+
+  # Generate versions.json
```

Comment:
Given that sharp builds wasm-vips with `--disable-bindings`, generating this `versions.json` file should probably not be done inside this block.
https://github.com/kleisauke/wasm-vips/blob/bdb02820bdbc2d509e641b89a0ba1967199ef57c/build.sh#L473

I think it would be better/clearer to place this after all those version numbers declarations.
https://github.com/kleisauke/wasm-vips/blob/bdb02820bdbc2d509e641b89a0ba1967199ef57c/build.sh#L163
(which hopefully also ensures that I don't forget to change this after a new dependency is added)